daily_milestone.go

```diff
@@ -95,32 +93,32 @@ func getProjectID(baseURL string, Token string, Projectname string, Namespace st
 		logger.Println("fail to read response data")
 
 	}
-	json.Unmarshal(respByte, &project)
+	json.Unmarshal(respByte, &projects)
 	defer resp.Body.Close()
-	for _, p := range project {
+	for _, p := range projects {
+		// To check for message404 error and that project is not forund
 		if p.Name == "message" {
 			fmt.Println(p.Name)
 		}
 
-		if p.Name == Projectname && p.NameSpace.Path == Namespace {
+		if p.Name == projectname && p.NameSpace.Path == namespace {
 			fmt.Println(strconv.Itoa(p.ID))
 			return strconv.Itoa(p.ID), nil
 		}
 	}
-	return "", fmt.Errorf("project %s not found", Projectname)
+	return "", fmt.Errorf("project %s not found", projectname)
 }
 
-// It is getting milestones data from the milestone API
+// It is getting milestones data from the milestone API and returning list of closed milestones
```

Comment:
Is it really returning the closed milestones? Check the code again. And try to understand the reasoning on what the check for closed or not is for.

Reply:
oh just checked it again. It is returning active milestones, right? the check is for !closed?

Reply:
Yes. The reasoning is that all closed milestones are most likely irrelevant and in the past so we don't actually need to compare the whole history. Only need to compare the currently open ones with the ones we want to create.